.github/workflows/dependencies-and-licences.yml

```diff
@@ -0,0 +1,59 @@
+name: Auto Update - Spec
+on:
+  workflow_dispatch:
+defaults:
+  run:
+    shell: bash
+jobs:
+  generate-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Core Repo
+        uses: actions/checkout@v2
+        with:
+          path: 'keptn/keptn'
+      - name: Checkout go-utils Repo
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.KEPTN_BOT_TOKEN }}
+          path: 'keptn/go-utils'
+          repository: 'keptn/go-utils'
+      - name: Checkout kubernetes-utils Repo
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.KEPTN_BOT_TOKEN }}
+          path: 'keptn/kubernetes-utils'
+          repository: 'keptn/go-utils'
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Install go-licence-detector
+        run: |
+         go get go.elastic.co/go-licence-detector
+      - name: generate-dependencies-and-licenses
+        run: |
+          TMP_DIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'keptndeps')
+          echo "storing dependencies files in ${TMP_DIR}"
+
+          MODULES="api approval-service cli configuration-service distributor helm-service jmeter-service lighthouse-service mongodb-datastore remediation-service secret-service shipyard-controller statistics-service"
+          for MODULE in $MODULES; do
+             echo "🔍 Analyzing dependencies in module $MODULE"
+             ( cd keptn/keptn/"$MODULE" || return ; go mod tidy > /dev/null 2>&1; go list -m -json all | go-licence-detector -depsTemplate=../.dependencies/templates/dependencies.csv.tmpl -depsOut="${TMP_DIR}"/"${MODULE}"-dependencies.txt  -overrides=../.dependencies/overrides/overrides.json )
+          done
+
+          echo "🔍 Analyzing dependencies in go-utils"
+          ( cd keptn/go-utils || return ; go mod tidy > /dev/null 2>&1; go list -m -json all | go-licence-detector -depsTemplate=../keptn/.dependencies/templates/dependencies.csv.tmpl -depsOut="${TMP_DIR}"/go-utils-dependencies.txt )
+
+          echo "🔍 Analyzing dependencies in kubernetes-utils"
+          ( cd keptn/kubernetes-utils || return ; go mod tidy > /dev/null 2>&1; go list -m -json all | go-licence-detector -depsTemplate=../keptn/.dependencies/templates/dependencies.csv.tmpl -depsOut="${TMP_DIR}"/kubernetes-utils-dependencies.txt )
+
+          cat "$TMP_DIR"/*.txt | sort | uniq > all-dependencies.txt
+
+          echo
+          echo "👍 done. written results to ./all-dependencies.txt"
+
+          cat all-dependencies.txt
```

Comment:
i think a good thing would be to attach the text file to the build as an artefact like it is done [here](https://github.com/keptn/keptn/blob/master/.github/workflows/integration_tests.yml#L727)